ui/src/stylesheets/specific/navigation.scss

```diff
@@ -115,9 +115,9 @@
 
 .nav-icon {
   margin-right: .5rem;
-  margin-bottom: 2px;
-  height: 1.1rem;
-  width: 1.1rem;
+  margin-bottom: 3px;
```

Comment:
This `margin-bottom` is a bit weird. :thinking: I would just remove it, what do you think?